tensorflow/lite/kernels/resize_bilinear_test.cc

```diff
@@ -465,5 +465,15 @@ TEST_P(ResizeBilinearOpTest, HorizontalResizeExtremeNegativeValuesInt16) {
 INSTANTIATE_TEST_SUITE_P(ResizeBilinearOpTest, ResizeBilinearOpTest,
                          testing::Values(TestType::kConst, TestType::kDynamic));
 
+TEST(ResizeBilinearOpTest_Negative, Int8MismatchedQuantizationFails) {
+  ResizeBilinearOpModel m({TensorType_INT8, {1, 2, 2, 1}, 0.0f, 0.0f, 0.5f, 1},
+                          {3, 3}, TestType::kConst);
+  TfLiteTensor* output_tensor = m.GetOutputTensor(0);
+  output_tensor->params.scale = 0.25f;
+  output_tensor->params.zero_point = 2;
+  m.SetInput<int8_t>({1, 2, 3, 4});
+  EXPECT_EQ(m.Invoke(), kTfLiteError);
```

Comment:
Can you actually check the contents of the error instead of just waiting for a generic error?